iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/CurrentLimitsTest.java

```diff
@@ -151,5 +151,64 @@ public void testSetterGetter() {
         assertTrue(temporaryLimit300.isFictitious());
         assertEquals(1400.0, temporaryLimit300.getValue(), 0.0);
         assertEquals(300, temporaryLimit300.getAcceptableDuration());
+
+        currentLimits.beginTemporaryLimit(300)
+                    .setName("5' bis")
+                    .setValue(1500.0)
+                    .setFictitious(false)
+                .endTemporaryLimit()
+                .beginTemporaryLimit(120)
+                    .setName("2'")
+                    .setValue(1550.0)
+                    .setFictitious(true)
+                .endTemporaryLimit();
+        CurrentLimits.TemporaryLimit temporaryLimit300bis = currentLimits.getTemporaryLimit(300);
+        assertEquals("5' bis", temporaryLimit300bis.getName());
+        assertFalse(temporaryLimit300bis.isFictitious());
+        assertEquals(1500.0, temporaryLimit300bis.getValue(), 0.0);
+        assertEquals(300, temporaryLimit300bis.getAcceptableDuration());
+        CurrentLimits.TemporaryLimit temporaryLimit120 = currentLimits.getTemporaryLimit(120);
+        assertEquals("2'", temporaryLimit120.getName());
+        assertTrue(temporaryLimit120.isFictitious());
+        assertEquals(1550.0, temporaryLimit120.getValue(), 0.0);
+        assertEquals(120, temporaryLimit120.getAcceptableDuration());
+    }
+
+    @Test
+    public void ensureNameUnicity() {
+        Line line = createNetwork().getLine("L");
+        CurrentLimits currentLimits = line.newCurrentLimits1()
+                .setPermanentLimit(100.0)
+                .beginTemporaryLimit()
+                    .setName("TL")
+                    .setAcceptableDuration(20 * 60)
+                    .setValue(1200.0)
+                    .ensureNameUnicity()
+                .endTemporaryLimit()
+                .beginTemporaryLimit()
+                    .setName("TL")
+                    .setAcceptableDuration(10 * 60)
+                    .setValue(1400.0)
+                    .ensureNameUnicity()
+                .endTemporaryLimit()
+                .beginTemporaryLimit()
+                    .setName("TL")
+                    .setAcceptableDuration(5 * 60)
+                    .setValue(1500.0)
+                    .ensureNameUnicity()
+                .endTemporaryLimit()
+                .add();
+
+        assertEquals("TL", currentLimits.getTemporaryLimit(20 * 60).getName());
+        assertEquals("TL#0", currentLimits.getTemporaryLimit(10 * 60).getName());
+        assertEquals("TL#1", currentLimits.getTemporaryLimit(5 * 60).getName());
+
+        currentLimits.beginTemporaryLimit(15 * 60)
```

Comment:
This is a strange design, why not just
currentLimits.getTemporaryLimit(15 * 60)
    .setName("TL")
    .setValue(1300.0);

Reply:
Design changed according to your suggestion. However, I am unsure if this is the good way to update `CurrentLimits` when adding a new `TemporaryLimit` (and not updating an existing one): currently, it creates a `TemporaryLimit` but does not add it directly as it is not valid (name and value are not immediately set). It is added when/if its name and value are correctly set. But it means that if I do: 
```java
currentLimits.getTemporaryLimitOrNew(10 * 60).setName("TL");
currentLimits.getTemporaryLimitOrNew(10 * 60).setValue(10);
```
no `TemporaryLimit` is created whereas if I do:
```java
currentLimits.getTemporaryLimitOrNew(10 * 60).setName("TL").setValue(10);
```
or 
```java
TemporaryLimit tl = currentLimits.getTemporaryLimitOrNew(10 * 60).setName("TL");
tl.setValue(10);
```
it is created. Is it okay ?
I specified when/if a `TemporaryLimit` is really added in debug messages but I am not sure it is enough.
On the other hand, I don't like the alternative to directly add a `TemporaryLimit` even if it is invalid.